app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    @like = Like.new
+    @like.author = current_user
+    @like.post = Post.find(params[:post_id])
+    if @like.save
+      redirect_to user_post_path(@like.post.author, @like.post)
+    else
+      render :new
+    end
+  end
+end
```

Comment:
- [ ] It would be better to create the `current_user ` method in `application_controller` so that you can use it. all controllers 
- right now when I try to create a comment or a like I get the following error please make sure to fix it so that it creates a comment and like without a prob 
<img width="1024" alt="Screenshot 2023-05-12 at 18 07 57" src="https://github.com/AbdimulhinYussuf3675/Blog-app/assets/61790587/35cc6b9c-6804-4360-8d10-5f784e3e7ad4">